### PR TITLE
Add to Playlist feature to Related and Channel lists

### PR DESCRIPTION
**Pull request by Shan11812**
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
Adds feature #1142

### CODE REVIEW

**Comment by mauriciocolli** [comment]
Thanks!

I think it's good to also add this to the `PlaylistFragment`, to cover the exact same use-case.

File: app/src/main/java/org/schabi/newpipe/fragments/list/channel/ChannelFragment.java [hunk 1]
```diff
@@ -183,6 +186,12 @@ public void onClick(DialogInterface dialogInterface, int i) {
                     case 4:
```
[the remark this answers is not in the document]
Comment (mauriciocolli) [reply app/src/main/java/org/schabi/newpipe/fragments/list/channel/ChannelFragment.java]: This constant values are simply the index of the choices array. It just would become a burden to maintain if someone change the order.

Also the array is directly above this fragment of code, so it shouldn't be a problem.

[the remark this answers is not in the document]
Comment (theScrabi) [reply app/src/main/java/org/schabi/newpipe/fragments/list/channel/ChannelFragment.java]: All right

**Comment by Shan11812** [comment]
> I think it's good to also add this to the PlaylistFragment, to cover the exact same use-case.
@mauriciocolli is it a good idea to add 'Add to Playlist' while already in a playlist? 
Also there are already 7 options to choose in that dialog.

**Comment by mauriciocolli** [comment]
@Shan11812 as I said previously, it is to cover the same use-case of the others, adding a single item into any playlist.

> Also there are already 7 options to choose in that dialog.

Yes, same situation with the channel fragment (6 choices, even their implementation will be a copy-paste of each other). If you have any suggestion of how to improve it, feel free to talk about it.

**Comment by theScrabi** [comment]
> I cri evrytiem

its not nice. We should try fixing it over here https://github.com/TeamNewPipe/NewPipe/pull/1308

**Pull request merged**